go.mod

```diff
@@ -3,6 +3,7 @@ module github.com/tektoncd/pipeline
 go 1.13
 
 require (
+	cloud.google.com/go/storage v1.6.0
```

Comment:
Any idea why this was added? It doesn't appear used anywhere.

Reply:
I gets added and removed as will (or random :sweat_smile:). That is one of the *high* mistery of go modules with k8s I feel :sweat_smile:

Reply:
no idea but leaves us with `git checkout -- go.mod` 😢